### ISSUE TITLE
Markup error in web client HOWTO

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4242]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4242|
|Type|defect|
|Created|2010-01-29 16:17:05Z|

In client.xhtml, line 180:

```
HTTP/1.0}
```

should be

```
HTTP/1.0
```

Attachments:

* [4242_markup_typo_fix.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/73e/73e062cc48878f84b76f57c811b601a146840e92/ff08eb02fe7f4c641a305ab0786a226ca532ca7a.patch) (753 bytes) - added by khorn on 2010-01-29 16:53:47Z - Typo fix

<details><summary>Searchable metadata</summary>

```
trac-id__4242 4242
type__defect defect
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1264781825000000 1264781825000000
changetime__1264796014000000 1264796014000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_1"></a>|@khorn removed owner|
|-|-|

Patch attached, please review.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra set owner to @khorn|
|-|-|

please merge, thanks.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_3"></a>|@khorn set status to `closed`|
|-|-|

(In [28089]) Merge: Fix typo in client.xhtml

Author: khorn
Reviewers: thijs
Fixes: [#4242](https://github.com/twisted/twisted/issues/4242)

Fix typo in /doc/web/howto/client.xhtml

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
